src/apis/tft/league/league.tft.ts

```diff
@@ -38,5 +38,31 @@ export class LeagueTFTApi extends BaseApiTft {
   public async getChallengerLeague (region: Regions) {
     return this.request<LeagueListDTO>(region, endpointsTFTV1.LeagueChallenger)
   }
-
+  /**
+   * Get league entries for given tier and division
+   * @param region
+   * @param tier
+   * @param division
+   * @param queue defaults to RANKED_TFT
+   * @param page defaults to 1
+   */
+  public async getByTierDivision (
+    region: Regions,
+    tier: Tiers,
+    division: Divisions,
+    queue: string = 'RANKED_TFT',
+    page: number = 1
+  ) {
+    const params = {
+      tier,
+      division
+    }
+    return this.request<LeagueEntryDTO[]>(
+      region,
+      endpointsTFTV1.LeagueByTierDivision,
+      params,
+      false,
```

Comment:
I referred to this part of the code. 
https://github.com/Sansossio/twisted/blob/1afe05df929c185184645cf226378d38c6225f86/src/apis/lol/league/league.ts#L52

My understanding is that `params` cannot be passed unless `forceError` is specified.